examples/src/ray_traced_triangle/mod.rs

```diff
@@ -0,0 +1,442 @@
+                    count: None,
+                },
+            ],
+        });
+
+        let vertices: [f32; 9] = [1.0, 1.0, 0.0, -1.0, 1.0, 0.0, 0.0, -1.0, 0.0];
+
+        let indices: [u32; 3] = [0, 1, 2];
+
+        let vertex_buffer = device.create_buffer_init(&BufferInitDescriptor {
+            label: Some("vertex buffer"),
+            contents: bytemuck::cast_slice(&vertices),
+            usage: BufferUsages::BLAS_INPUT,
+        });
+
+        let index_buffer = device.create_buffer_init(&BufferInitDescriptor {
+            label: Some("vertex buffer"),
+            contents: bytemuck::cast_slice(&indices),
+            usage: BufferUsages::BLAS_INPUT,
+        });
+
+        let blas_size_desc = BlasTriangleGeometrySizeDescriptor {
+            vertex_format: wgpu::VertexFormat::Float32x3,
+            // 3 coordinates per vertex
+            vertex_count: (vertices.len() / 3) as u32,
+            index_format: Some(IndexFormat::Uint32),
+            index_count: Some(indices.len() as u32),
+            flags: wgpu::AccelerationStructureGeometryFlags::OPAQUE,
+        };
+
+        let blas = device.create_blas(
+            &CreateBlasDescriptor {
+                label: None,
+                flags: AccelerationStructureFlags::PREFER_FAST_TRACE,
+                update_mode: AccelerationStructureUpdateMode::Build,
+            },
+            BlasGeometrySizeDescriptors::Triangles {
+                descriptors: vec![blas_size_desc.clone()],
+            },
+        );
+
+        let tlas = device.create_tlas(&CreateTlasDescriptor {
+            label: None,
+            max_instances: 3,
+            flags: AccelerationStructureFlags::PREFER_FAST_TRACE,
+            update_mode: AccelerationStructureUpdateMode::Build,
+        });
+
+        let mut tlas_package = TlasPackage::new(tlas);
+
+        tlas_package[0] = Some(TlasInstance::new(
+            &blas,
+            Mat4::from_translation(Vec3 {
+                x: 0.0,
+                y: 0.0,
+                z: 0.0,
+            })
+            .transpose()
+            .to_cols_array()[..12]
+                .try_into()
+                .unwrap(),
+            0,
+            0xff,
+        ));
+
+        tlas_package[0] = Some(TlasInstance::new(
+            &blas,
+            Mat4::from_translation(Vec3 {
+                x: -1.0,
+                y: -1.0,
+                z: -2.0,
+            })
+            .transpose()
+            .to_cols_array()[..12]
+                .try_into()
+                .unwrap(),
+            0,
+            0xff,
+        ));
+
+        tlas_package[0] = Some(TlasInstance::new(
+            &blas,
+            Mat4::from_translation(Vec3 {
+                x: 1.0,
+                y: -1.0,
+                z: -2.0,
+            })
+            .transpose()
+            .to_cols_array()[..12]
+                .try_into()
+                .unwrap(),
+            0,
+            0xff,
+        ));
```

Comment:
typo: all of these have index 0, which is causing the test to fail.

Reply:
ah, thats probably my fault just when changing to index_mut